docs/openapi/schemas/VerificationResult.yml

```diff
@@ -3,15 +3,23 @@ type: object
 properties:
   verified:
     type: boolean
-  checks:
+  verifications:
     type: array
     items:
-      type: string
-  warnings:
-    type: array
-    items:
-      type: string
-  errors:
-    type: array
-    items:
-      type: string
+      $ref: ./Verification.yml
+example: 
```

Comment:
Should `verified` and `verifications` be required?

Reply:
imo verified yes, verifications no.